kadi/commands/tests/test_commands.py

```diff
@@ -1329,3 +1329,48 @@ def test_get_rltt_scheduled_stop_time():
     cmds = commands.get_cmds("2023:009:12:00:00", "2023:010")
     assert cmds.get_rltt() is None
     assert cmds.get_scheduled_stop_time() is None
+
+
+# For HRC not run testing
+stop_date_2023_12_20 = stop_date_fixture_factory("2023-12-20")
+
+
+@pytest.mark.skipif(not HAS_INTERNET, reason="No internet connection")
+def test_hrc_not_run_scenario(stop_date_2023_12_20):  # noqa: ARG001
+    """Test custom scenario with HRC not run"""
+    from kadi.commands.states import get_states
+
+    # First make the cmd_events.csv file for the scenario
+    scenario = "hrc_not_run"
+    cmds_dir = Path(commands_v2.conf.commands_dir) / scenario
+    cmds_dir.mkdir(exist_ok=True, parents=True)
+    # Note variation in format of date, since this comes from humans.
+    cmd_evts_text = """\
```

Comment:
This test would probably be stronger if the states were evaluated with and without the hrc not run -- but as a sanity and regress test this is good.

Reply:
Good idea, I'll fix this.